packages/lit-dev-content/site/docs/localization/runtime-mode.md

```diff
@@ -0,0 +1,304 @@
+---
+title: Runtime localization mode
+eleventyNavigation:
+  key: Runtime mode
+  parent: Localization
+  order: 2
+---
+
+In Lit Localize runtime mode, one JavaScript or TypeScript module is generated
+for each of your locales. Each generated module contains the localized templates
+for that locale. When your application switches locales, the module for that
+locale is imported, and all components are re-rendered.
+
+See [output modes](/docs/localization/overview/#output-modes) for a comparison
+of Lit Localize output modes.
+
+#### Example output
+
+```js
+// locales/es-419.ts
+export const templates = {
+  h3c44aff2d5f5ef6b: html`Hola <b>Mundo!</b>`,
+};
+```
+
+## Example of using runtime mode
+
+The following example demonstrates an application built with Lit Localize
+runtime mode:
+
+{% playground-example "docs/libraries/localization/runtime" "x-greeter.ts" %}
+
+For a full working example of runtime mode that you can use as a template, see
+the
+[JavaScript](https://github.com/lit/lit/tree/main/packages/localize/examples/runtime-js)
+and
+[TypeScript](https://github.com/lit/lit/tree/main/packages/localize/examples/runtime-ts)
+directories in the Lit GitHub repo.
```

Comment:
I tripped over the phrasing of this, which made it sound to me like there were top-level `JavaScript` and `TypeScript` directories in the Lit repo that somehow served as localization examples. Suggested rephrasing:

"The `@lit-labs/localize` source in the Lit GitHub repo includes full, working examples ([JavaScript](https://github.com/lit/lit/tree/main/packages/localize/examples/runtime-js), [TypeScript](https://github.com/lit/lit/tree/main/packages/localize/examples/runtime-ts)) that you can use as templates."